example/lib/.gitignore

```diff
@@ -0,0 +1 @@
+api_keys.dart
```

Comment:
Make sure people don't accidentally commit `api_keys.dart` if they use the example app as reference